### PR TITLE
Features/crud articles

### PR DESCRIPTION
**Pull request by Ernestoale98**
CRUD of table articles done @cesarureno

### CODE REVIEW

File: app/Http/Controllers/CategoriaController.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    public function update(Request $request)
+    {
+        if (!$request->ajax()) return redirect('/');
+        $categoria = Categoria::findOrFail($request->id);
+        $categoria->fill($request->all());
+        $categoria->condicion = '1';
+        $categoria->save();
+    }
+
+    public function desactivar(Request $request)
```
Comment (cesarureno) [inline app/Http/Controllers/CategoriaController.php]: Please change `desactivar` and `activar` function to only one function

Comment (Ernestoale98) [reply app/Http/Controllers/CategoriaController.php]: Ok I will check it now,thanks

**Comment by Ernestoale98** [comment]
functions activar and desactivar united in one @cesarureno

**Comment by Ernestoale98** [comment]
conflicts fixed ready to pull request @cesarureno

**Pull request merged**